topology/Makefile.am

```diff
@@ -19,6 +19,7 @@ DEPS = \
 MACHINES = \
 	sof-cht-nocodec.tplg \
 	sof-cht-max98090.tplg \
+	sof-apl-hdmi.tplg \
 	sof-apl-nocodec.tplg \
```

Comment:
I am not really hot on this one. I know Liam merged it so I guess it's a discussion for a follow-up patch...
1. why does it have any references to apl?
2. If the goal is to use the standard hdaudio machine driver, then why is this necessary? The analog links will only be used if the hdaudio coded is detected, so something is very odd here.

Reply:
@plbossart It's for testing HDMI audio on UP2.